<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolRunTool</class>
 <widget class="QToolBox" name="PolRunTool">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ToolBox</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="currentIndex">
   <number>2</number>
  </property>
  <property name="tabSpacing">
   <number>1</number>
  </property>
  <widget class="QWidget" name="page_Desciption">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>431</height>
    </rect>
   </property>
   <attribute name="label">
    <string>Description</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Author:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Author">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="text_Description">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_Options">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>431</height>
    </rect>
   </property>
   <attribute name="label">
    <string>Options</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="layout_Options">
      <property name="spacing">
       <number>6</number>
      </property>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_Parameters">
   <attribute name="label">
    <string>Parameters</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="layout_Parameters">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
